sdk/eventhub/event-hubs/test/iothub.spec.ts

```diff
@@ -12,7 +12,9 @@ import { EnvVarKeys, getEnvVars } from "./utils/testUtils";
 const env = getEnvVars();
 
 describe("EventHub Client with iothub connection string ", function(): void {
-  const service = { connectionString: env[EnvVarKeys.IOTHUB_CONNECTION_STRING] };
+  const service = {
+    connectionString: (env[EnvVarKeys.IOTHUB_CONNECTION_STRING] as string) || ""
```

Comment:
Have we run the tests in CI?
I wonder if this env value is using the event hub compatible connection string or not

Reply:
No, only locally. I need to find out how to actually change it for the CI system :)